.github/workflows/nix-build.yml

```diff
@@ -0,0 +1,29 @@
+name: nix build
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
maybe you would like to set `0.0.x` here because main is gonna be off for a while

Reply:
done